src/selector/Selector.tsx

```diff
@@ -12,15 +12,16 @@ export function Selector({ nodes: rootNodes, children }: Props) {
     const renderers = createComponentsRenderersFromChildren(children);
 
     function renderNodes(nodes: Node[]): ReactNode {
-        return nodes.map((node, index) => <Fragment key={index}>{renderNode(node)}</Fragment>);
+        return nodes.map((node, index) => renderNode(node, index));
```

Comment:
This change was needed so we can clone children in `Table` and `TableRow` without having to worry about them being wrapped in `Fragment`.